README.md

```diff
@@ -51,4 +51,4 @@ The Unison Driver is licensed under the BSD3 license, see the
 
 ## Further Reading
 
-Check [the Unison website](https://unison-code.github.io/).
\ No newline at end of file
+Check [the Unison website](https://unison-code.github.io/).
```

Comment:
Spurious change, please remove it.

Reply:
Actually I am not sure why it is reading this as a change

Reply:
Probably your editor added a newline at the end of the file or something like that, please have a look.

Reply:
yes, most probably, I am looking into it now

Reply:
Problem solved! my editor (gedit) added a new line for an unknown reason. However, I removed it in the last commit "newline truncated".